api/utils/download.py

```diff
@@ -29,16 +29,25 @@ def parse_args():
 
 def download_models(models: List[str]):
     for model in models:
+        if os.path.isfile(f"weights/ggml-alpaca-{model}-q4_0.bin"):
```

Comment:
This shouldnt be a hardcoded path/name. That will limit future changes.

Reply:
The weights folder is located at `/usr/src/app/weights`.

Reply:
> This shouldnt be a hardcoded path/name. That will limit future changes.
Isnt this the same approach being used on line 53?
 os.rename(
                "/usr/src/app/weights/ggml-model-q4_0.bin", f"/usr/src/app/weights/**ggml-alpaca-{model}-q4_0.bin"**
            )

Reply:
> The weights folder is located at `/usr/src/app/weights`.
The path was working on my tests, I will double check it